app/controllers/api/base_controller/authentication.rb

```diff
@@ -30,6 +30,10 @@ def require_api_user_or_token
         log_api_auth
       end
 
+      def api_config
+        @api_config ||= VMDB::Config.new("vmdb").config[Settings.base.module.to_sym] || {}
```

Comment:
If you're changing this, might as well use the new global Settings directly, and since you are doing that, there's no need to memoize since the Global Settings are in memory...

```ruby
def api_config
  ::Settings[Settings.base.module]
end
```

P.S. *THIS* I why I think it's confusing to have two same-named classes.